api/v1/client/src/commonMain/kotlin/OrtServerClient.kt

```diff
@@ -59,7 +59,7 @@ class OrtServerClient(
 
             val client = createOrtHttpClient(JSON) {
                 defaultRequest {
-                    url(config.baseUrl)
+                    url(config.baseUrl.takeIf { it.endsWith("/") } ?: "${config.baseUrl}/")
```

Comment:
Any chance to have this backed by a test with reasonable effort?

Reply:
I tried, but I don't think that testing this is possible without changing the place where this fix is applied.

What I tried:
* Mocking the client in order to verify the call to the correct. Not possible because a client instance is created in lin 60, therefore it cannot be mocked
* Accessing the `defaultRequest` parameters and checking if the slash is correctly applied: I didn't find a way how to do that.

Do you have any other ideas how this could be tested?
Or does this mean that maybe we should apply the fix at a different place?

Reply:
Not sure, but wouldn't a test be possible on a higher level for code that is using the client? So, sort of an integration test?

Reply:
Moved the change to the CLI to ensure the URL is always the same, which also makes testing simpler.